projects/1.19/assets/miskatonic-mysteries/miskatonicmysteries/lang/zh_cn.json

```
@@ -0,0 +1,507 @@
+	"miskatonicmysteries.command.remove_medium": "%s学习了%d次",
+	"miskatonicmysteries.command.remove_medium.self": "学习了%d次",
+	"miskatonicmysteries.command.remove_sanity_expansion": "移除了%s的理智扩张，从%s",
+	"miskatonicmysteries.command.remove_sanity_expansion.failure": "%s不存在，作为%s的理智扩张",
+	"miskatonicmysteries.command.removed_spell": "%s：移除了%d次",
+	"miskatonicmysteries.command.removed_spell.self": "成功移除了%d次",
+	"miskatonicmysteries.command.set_ascension": "设置%s的在第%d阶段前往%s的路线",
+	"miskatonicmysteries.command.set_ascension.self": "设置在第%d阶段前往%s的路线",
+	"miskatonicmysteries.command.set_max_spells": "设置%s的咒语上限为%d",
+	"miskatonicmysteries.command.set_max_spells.self": "设置自己的咒语上限为%d",
+	"miskatonicmysteries.command.set_power_pool": "设置%s的能量储值为%d",
+	"miskatonicmysteries.command.set_power_pool.self": "设置自己的能量储值为%d",
+	"miskatonicmysteries.command.set_sanity": "设置%s的理智等级为%d",
+	"miskatonicmysteries.command.set_sanity.self": "设置自己的理智等级为%d",
+	"miskatonicmysteries.command.set_spell_max": "设置%s的咒语上限为%d",
+	"miskatonicmysteries.command.set_spell_max.self": "设置自己的咒语上限为%d",
+	"miskatonicmysteries.command.stats": "展示在密斯卡托尼克之谜中玩家%s的统计数字：",
+	"miskatonicmysteries.command.summon_investigator": "成功的为%s召唤了一位研究员",
+	"miskatonicmysteries.command.summon_investigator.failure": "无法召唤研究员",
+
+	"miskatonicmysteries.gui.blessings": "祝福",
+	"miskatonicmysteries.gui.configure_predicates": "配置谓词",
+	"miskatonicmysteries.gui.spell_effects": "状态效果",
+	"miskatonicmysteries.gui.spell_mediums": "媒介",
+	"miskatonicmysteries.gui.spell_select": "咒语选择",
+	"miskatonicmysteries.gui.sudoku": "数独",
+	"miskatonicmysteries.gui.config_mode.players": "仅限玩家",
+	"miskatonicmysteries.gui.config_mode.mobs": "仅限生物",
+	"miskatonicmysteries.gui.config_mode.all": "全部",
+	"miskatonicmysteries.gui.exclude_players_query": "效果排除？",
+
+	"miskatonicmysteries.necronomicon.landing": "永世长眠者未必死亡，$(br)奇迹降临时，死亡亦将死亡。",
+	"miskatonicmysteries.necronomicon.subtitle": "一本载有死者名字的书",
+	"miskatonicmysteries.science_journal.landing": "本书仅由密斯卡托尼克大学及其附属大学，公司等机构所有。$(br2)其他非附属的个人或机构持有本书将会受到法律的制裁。",
+	"miskatonicmysteries.science_journal.subtitle": "第二版",
+
+	"rei.miskatonicmysteries.chemistry_recipe": "非法化学",
+	"rei.miskatonicmysteries.octagram_rite": "神秘魔法",
+	"rei.miskatonicmysteries.tooltip.primable": "蓄势待发",
+	"rite.miskatonicmysteries.broken_veil": "破损的面纱",
+	"rite.miskatonicmysteries.burned_veil": "烧焦的面纱",
+	"rite.miskatonicmysteries.damage": "学习破坏",
+	"rite.miskatonicmysteries.enchant_canvas": "附魔画",
+	"rite.miskatonicmysteries.golden_flock": "黄金入教礼",
+	"rite.miskatonicmysteries.harrows": "学习折磨",
+	"rite.miskatonicmysteries.hastur_simulacrum": "卡尔克萨的幻影",
+	"rite.miskatonicmysteries.hysteria": "歇斯底里",
+	"rite.miskatonicmysteries.knockback": "学习击退",
+	"rite.miskatonicmysteries.masterpiece": "创造杰作",
+	"rite.miskatonicmysteries.mourning_dead": "悼念死者",
+	"rite.miskatonicmysteries.regeneration": "学习治愈",
+	"rite.miskatonicmysteries.revert_biome": "净化大千世界",
+	"rite.miskatonicmysteries.resistance": "学习反抗",
```

```suggestion
	"rite.miskatonicmysteries.resistance": "学习抗性",
```
游戏内是什么效果
这里改则下方spell_effect也要改